std_msgs/Header header
float32 front
float32 front_right
float32 right
float32 front_left
float32 left
float32 rear
float32 recommended_speed
int32 recommended_steering
string action
